dockerfiles/init/modules/openshift/files/scripts/deploy_che.sh

```diff
@@ -428,13 +435,21 @@ cat "${CHE_DEPLOYMENT_FILE_PATH}" | \
     oc apply --force=true -f -
 else
   echo "[CHE] Deploying Che on OpenShift Container Platform (image ${CHE_IMAGE})"
-  curl -sSL http://central.maven.org/maven2/io/fabric8/tenant/apps/che/"${OSIO_VERSION}"/che-"${OSIO_VERSION}"-openshift.yml | \
+  DEFAULT_CHE_DEPLOYMENT_FILE_PATH=./che-spi-openshift.yml
+  CHE_DEPLOYMENT_FILE_PATH=${CHE_DEPLOYMENT_FILE_PATH:-${DEFAULT_CHE_DEPLOYMENT_FILE_PATH}}
+
+cat "${CHE_DEPLOYMENT_FILE_PATH}" | \
     if [ ! -z "${OPENSHIFT_NAMESPACE_URL+x}" ]; then sed "s/    hostname-http:.*/    hostname-http: ${OPENSHIFT_NAMESPACE_URL}/" ; else cat -; fi | \
     sed "s/          image:.*/          image: \"${CHE_IMAGE_SANITIZED}\"/" | \
     sed "s/          imagePullPolicy:.*/          imagePullPolicy: \"${IMAGE_PULL_POLICY}\"/" | \
     sed "s|    keycloak-oso-endpoint:.*|    keycloak-oso-endpoint: ${KEYCLOAK_OSO_ENDPOINT}|" | \
     sed "s|    keycloak-github-endpoint:.*|    keycloak-github-endpoint: ${KEYCLOAK_GITHUB_ENDPOINT}|" | \
     sed "s/    keycloak-disabled:.*/    keycloak-disabled: \"${CHE_KEYCLOAK_DISABLED}\"/" | \
+    sed "s|    CHE_INFRA_OPENSHIFT_PROJECT:.*|    CHE_INFRA_OPENSHIFT_PROJECT: ${CHE_INFRA_OPENSHIFT_PROJECT}|" | \
```

Comment:
is these lines needed only for ocp flavor? not for minishift or osio ?

Reply:
For `minishift` and `osio` these lines already are there

Reply:
@sleshchenko I guess for osio flavor `OAUTH__TOKEN` / `USERNAME` / `PASSWORD` vars are not set deliberately, so that default service account token would be used for object creation in `-che` namespace, right ?

Reply:
![KEWL](https://pbs.twimg.com/profile_images/124284534/color_bus_buddy.jpg)

Reply:
@ibuziuk you're right. I think it is useful only for `OCP` and `minishift` flavors. So, now for OSIO service account will be used for objects creation in the same project. And later, I believe, we'll implement using of users' OpenShift tokens to create workspaces' object in different projects.
I am not sure how it will work for `OCP` and `minishift` but now it is implemented by using user token or credentials.

Reply:
sounds good. Tested on osio and all works just fine. +1 to apply